### PR TITLE
MCPERSON-2822 add file icons for message center attachments

### PR DESCRIPTION
**Pull request by orteth01**
added the following icons:

file-doc
image

file-excel
image

file-pdf
image

### CODE REVIEW

**Comment by orteth01** [comment]
@hellatan why is the `dist` directory in `.gitignore`? I ran `yarn build` but that directory isn't included...

**Comment by orteth01** [comment]
> why is the dist directory in .gitignore? I ran yarn build but that directory isn't included...

@hellatan nvm... https://github.com/1stdibs/dibs-vg/commit/99d274db44990d38d06efce42d5b3018c8d35d94

**Comment by brettjashford** [comment]
looks fine to me, but i don't know much about the svg process. assigning to dale to merge.

File: src/file-doc.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg version="1.1" class="dibsvg-svg dibsvg-file-doc" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 250 250">
+	<path fill="#000000" d="M148.2,0H30v250h189V73.7L148.2,0z M148,12.8L203,70h-55V12.8z M210,241H39V9h100v70h71V241z"/>
+	<polygon fill="#000000" points="114.7,117 101.3,173.1 99.6,173.2 87.7,117 71,117 90.4,206 108.1,206 122.1,153.3 123.8,153.3
```
Comment (hellatan) [inline src/file-doc.svg]: remove both of these `fill` attributes and remove the whitespace at the end of this `points` attribute

File: src/file-excel.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<svg version="1.1" class="dibsvg-svg dibsvg-file-excel" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 250 250">
+	<path fill="#000000" d="M150.2,0H32v250h189V73.7L150.2,0z M150,12.8L205,70h-55V12.8z M212,241H41V9h100v70h71V241z"/>
+	<polygon fill="#000000" points="100.5,104.5 78.2,104.5 113.9,160.3 78.2,216.1 100.5,216.1 125.1,173.7 149.7,216.1 172,216.1
```
Comment (hellatan) [inline src/file-excel.svg]: same here re: remove fill, remove whitespace

File: src/file-pdf.svg [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<svg version="1.1" class="dibsvg-svg dibsvg-file-pdf" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 250 250">
+	<path fill="#000000" d="M148.2,0H30v250h189V73.7L148.2,0z M148,12.8L203,70h-55V12.8z M210,241H39V9h100v70h71V241z"/>
+	<path fill="#000000" d="M123.4,124c0,0,4.3-24.2,3.1-32.4c-0.2-1.1-0.3-1.4-0.6-2.3l-0.4-0.9c-1.2-2.7-3.4-5.5-7-5.3l-2.1-0.1
```
Comment (hellatan) [inline src/file-pdf.svg]: remove `fill`

**Comment by orteth01** [comment]
@hellatan yea wasn't sure about those `fill` properties haha. removed them 👍

**Comment by hellatan** [comment]
@orteth01 - https://github.com/1stdibs/dibs-vg/blob/master/CONTRIBUTING.md

**Comment by orteth01** [comment]
@hellatan yea I followed the contributing guide but wasn't sure because of this line https://github.com/1stdibs/dibs-vg/blame/master/CONTRIBUTING.md#L9.

the svg's i got back from the designer had `fill="#010101`, which I thought was weird so I changed it to `fill="#000000`. didn't realize it could be removed completely.

**Comment by hellatan** [comment]
>  didn't realize it could be removed completely.

I guess we should add more attributes that can be removed here: https://github.com/1stdibs/dibs-vg/blob/master/CONTRIBUTING.md#for-developers cc @nameofname

**Comment by hellatan** [comment]
@orteth01 ⬇️ 〰️ tasks, please

**Comment by orteth01** [comment]
> I guess we should add more attributes that can be removed here: https://github.com/1stdibs/dibs-vg/blob/master/CONTRIBUTING.md#for-developers

yea, it mentions the only attributes allowed on the `<svg />` element [here](https://github.com/1stdibs/dibs-vg/blame/master/CONTRIBUTING.md#L49-L51). maybe something similar for the interior elements would help to clarify.

**Comment by nameofname** [comment]
@orteth01 good point - I'm accepting PRs...

**Pull request merged**